core/opendaq/signal/src/dimension_impl.cpp

```diff
@@ -173,7 +173,7 @@ ErrCode DimensionImpl::equals(IBaseObject* other, Bool* equals) const
 ListPtr<IBaseObject> DimensionImpl::getLinearLabels() const
 {
     const SizeT size = rule.getParameters().get("size");
-    const int delta = rule.getParameters().get("delta");
+    const Float delta = rule.getParameters().get("delta");
```

Comment:
Should this be Float? In our documentation we state we only allow integer steps.